modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -250,10 +245,15 @@ test('Check that the Dynamic Segment does not continue to appear in the audience
 	});
 
 	await test.step('Check that the created segment appears on the Audience card', async () => {
-		await expectToBeVisible({
-			itemNames: segmentsName,
-			page,
-		});
+		const itemNamesArray = Array.isArray(segmentsName)
+			? segmentsName
+			: [segmentsName];
```

Comment:
Hey @fmarcio, I think it is not necessary, right? Because we know that the segmentsName variable is an array. What do you think?

Reply:
done